### PR TITLE
Removing old parameters.

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
Working on bug fixes to TimerProxy.

## PR Checklist
* [ ] Applies to work item: #2397
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

File: src/ApiService/ApiService/onefuzzlib/Extension.cs [hunk 1]
```diff
@@ -136,13 +136,6 @@ private class Settings {
     }
 
     public static VMExtensionWrapper AzMonExtension(AzureLocation region, AzureMonitorExtensionConfig azureMonitor) {
```
Comment (stishkin) [inline src/ApiService/ApiService/onefuzzlib/Extension.cs:138]: you can test this out without submitting PR can't you ?

Comment (stishkin) [reply src/ApiService/ApiService/onefuzzlib/Extension.cs:138]: or you validated and now this is a good change ?

**Comment by stishkin** [comment]
is this a change that fixes the issue ?

File: src/ApiService/ApiService/onefuzzlib/Extension.cs (hunk 1 above)
Comment (nharper285) [reply src/ApiService/ApiService/onefuzzlib/Extension.cs:138]: You're right. I don't need to submit PR just yet.

**Pull request closed without merging**